packages/js/src/feeds/feeds.ts

```diff
@@ -59,35 +70,120 @@ export class Feeds extends BaseModule {
         this._emitter.emit('feeds.fetch_count.error', { args, error });
         throw error;
       }
     });
   }
 
+  async read(args: InstanceArgs): Promise<Notification>;
+  async read(args: ReadArgs): Promise<Notification> {
+    return this.callWithSession(async () =>
+      markRead({
+        emitter: this._emitter,
+        apiService: this._inboxService,
+        args,
+      })
+    );
+  }
+
+  async unread(args: InstanceArgs): Promise<Notification>;
+  async unread(args: UnreadArgs): Promise<Notification> {
+    return this.callWithSession(async () =>
+      markUnread({
+        emitter: this._emitter,
+        apiService: this._inboxService,
+        args,
+      })
+    );
+  }
+
+  async archived(args: InstanceArgs): Promise<Notification>;
+  async archived(args: ArchivedArgs): Promise<Notification> {
+    return this.callWithSession(async () =>
+      markArchived({
+        emitter: this._emitter,
+        apiService: this._inboxService,
+        args,
+      })
+    );
+  }
+
+  async unarchived(args: InstanceArgs): Promise<Notification>;
+  async unarchived(args: UnarchivedArgs): Promise<Notification> {
+    return this.callWithSession(async () =>
+      markUnarchived({
+        emitter: this._emitter,
+        apiService: this._inboxService,
+        args,
+      })
+    );
+  }
+
+  async readAll({ tags }: { tags?: NotificationFilter['tags'] } = {}): Promise<void> {
+    return this.callWithSession(async () => {
+      try {
+        this._emitter.emit('feeds.read_all.pending', { args: { tags } });
+
+        await this._inboxService.readAll({ tags });
+
+        this._emitter.emit('feeds.read_all.success', { args: { tags }, result: undefined });
+      } catch (error) {
+        this._emitter.emit('feeds.read_all.error', { args: { tags }, error });
+        throw error;
+      }
+    });
+  }
+
+  async archivedAll({ tags }: { tags?: NotificationFilter['tags'] } = {}): Promise<void> {
+    return this.callWithSession(async () => {
+      try {
+        this._emitter.emit('feeds.archived_all.pending', { args: { tags } });
+
+        await this._inboxService.archivedAll({ tags });
+
+        this._emitter.emit('feeds.archived_all.success', { args: { tags }, result: undefined });
+      } catch (error) {
+        this._emitter.emit('feeds.archived_all.error', { args: { tags }, error });
+        throw error;
+      }
+    });
+  }
+
+  async readArchivedAll({ tags }: { tags?: NotificationFilter['tags'] } = {}): Promise<void> {
+    return this.callWithSession(async () => {
+      try {
+        this._emitter.emit('feeds.read_archived_all.pending', { args: { tags } });
+
+        await this._inboxService.readArchivedAll({ tags });
+
+        this._emitter.emit('feeds.read_archived_all.success', { args: { tags }, result: undefined });
+      } catch (error) {
+        this._emitter.emit('feeds.read_archived_all.error', { args: { tags }, error });
+        throw error;
+      }
+    });
+  }
+
+  /**
+   * @deprecated
```

Comment:
Let's remove the methods, there is no need for deprecation as this is the first version of the SDK.

Reply:
I'll do that in another PR